internal/rest/avi_obj_vsvip.go

```diff
@@ -124,8 +125,9 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, vsCach
 	} else {
 		var vips []*avimodels.Vip
 		vip := avimodels.Vip{
-			VipID:          &vipId,
-			AutoAllocateIP: &autoAllocate,
+			VipID:                  &vipId,
+			AutoAllocateIP:         &autoAllocate,
+			AutoAllocateFloatingIP: &vsvip_meta.EnablePublicIP,
```

Comment:
does this API through an error if you are doing this on the vCenter cloud? Or it just assumes that the feature isn't supported?